src/templates/forum/index.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'machina/forum/index.html' %}
+
+{% block content %}
+  <div class="d-flex justify-content-end">
+    <a class="btn btn-primary right-0"
+       href="{% url 'forums:forum-create' %}">Create new forum</a>
```

Comment:
This is much better placement!

Reply:
I think the placement of this button is fine, considering it's a separate admin function, but it's worth considering that it is inconsistent with the placement of similar buttons in the django-machina UI.
![Screenshot 2023-11-30 at 11 07 36 AM](https://github.com/uchicago-cs/chigame/assets/59244307/fbcb141e-5dc3-4fb1-ac57-005e0a088807)
![Screenshot 2023-11-30 at 11 05 55 AM](https://github.com/uchicago-cs/chigame/assets/59244307/cded067d-360b-4055-bb71-c06ee300745c)

Also, the button is visible even if you are logged out. It should only be visible after you log in as an admin.
![Screenshot 2023-11-30 at 11 09 01 AM](https://github.com/uchicago-cs/chigame/assets/59244307/9b49db0d-1571-4888-8ca1-6d96b25fc31b)

Reply:
I could move it to the top left. That would make the most sense in my opinion. Also, thanks for pointing out the admin issue. I'll change that.